avatars/avatars.js

```diff
@@ -1935,6 +1943,91 @@ class Avatar {
 
     // this.springBoneTimeStep.update(timeDiff);
     this.springBoneManager && this.springBoneManager.lateUpdate(timeDiffS);
+   
+    const _applyWind = () => {
+      const winds = metaversefile.getWinds();
+      //console.log(winds)
+      const timeS = performance.now() / 1000;
```

Comment:
This can be read from a `timestamp` passed in from `update()`.

In general we should not do `performance.now()` in our code so the timestamps for all apps are consistent, rather than slightly off.

Reply:
used the timestamp from avatar.js